omnimatter_energy/info.json

```diff
@@ -13,6 +13,7 @@
     "(?)bobplates",
     "?boblogistics",
     "(?)deadlock-integrations",
-    "?bobassembly"
+    "?bobassembly",
+    "!aai-industry"
 ]
```

Comment:
Does info.json support trailing commas on the last line? Would make diffs easier to read.

Reply:
Nope, besides the VS errors factorio does not load that mod at all if you add a comma there